tests/unit/controllers/notificationControllerTest.js

```diff
@@ -72,13 +73,14 @@ describe('controller: NotificationController', function () {
 
     describe('Are the scope methods working as expected', function () {
         it('resetNotifications should reset notifications', function () {
-            var notification;
-            scope.notificationData = null;
+            var notification = new Notification();
```

Comment:
```suggestion
            const notification = new Notification();
```
Possible improvement.
I don't know if we should change this or not since.
It is convenient to change because the line is being changed anyway.

However, The rest of the file uses `var` in similar ways so changing this in isolation would be inconsistent.

Feel free to ignore this request if you feel following consistency to be important here.

Updating the other similar cases is also an option.

Reply:
Again. Same here. Refactoring var to const should be a dedicated task for code maintenance which applies a linting rule and such. I still consider this scope creep regardless of maybe consistency rule doesn't apply to new code.